src/Features/Core/Portable/Diagnostics/EngineV1/DiagnosticIncrementalAnalyzer.AnalyzerExecutor.cs

```diff
@@ -94,15 +95,28 @@ public AnalyzerExecutor(DiagnosticIncrementalAnalyzer owner)
                     {
                         var memberDxData = await GetSemanticDiagnosticsAsync(analyzerDriver, stateSet.Analyzer).ConfigureAwait(false);
 
-                        diagnosticData = _owner.UpdateDocumentDiagnostics(existingData, ranges.Ranges, memberDxData.AsImmutableOrEmpty(), root.SyntaxTree, member, memberId);
-                        ValidateMemberDiagnostics(stateSet.Analyzer, document, root, diagnosticData);
+                        // if all the current diagnostics have all their locations in this document,
```

Comment:
hmmm.. this code seems changed quite a bit over what I remember. anyway, v2 engine doesn't make span based analysis anymore so it shouldn't be an issue for v2.